src/mocks/datapoints.ts

```diff
@@ -33,31 +33,42 @@ export interface MockChartDataConfig {
   positivePeakPoints?: number;
 }
 
-export const getMockDatapointsClientObject = (config?: any) => {
-  return {
-    retrieve: async (query: any): Promise<DatapointsArray> => {
-      const { granularity, start, end, id } = query.items[0];
-      const result = getRandomdata(
-        id,
-        (start && +start) || 0,
-        (end && +end) || 0,
-        100,
-        granularity ? getGranularityInMS(granularity) : undefined,
-        config
-      );
-      return [result];
-    },
-  };
+const getMockDatapointsRetrieve = (
+  config?: MockChartDataConfig[] | undefined
+) => async (
+  query: any
+  // tslint:disable-next-line:no-identical-functions
```

Comment:
you should be really confident in what you are doing every time you disable ts-lint, it's here on purpose after all